modules/infra/digitalocean/main.tf

```diff
@@ -106,5 +147,9 @@ resource "digitalocean_firewall" "k8s_cluster" {
     destination_addresses = ["0.0.0.0/0", "::/0"]
   }
 
-  depends_on = [digitalocean_droplet.droplet]
+  depends_on = [
```

Comment:
Need to figure out how this behaves when the conditional is introduced.

Reply:
I'm going to remove these dependencies. After thinking it over, they seem unnecessary as the firewall rules don't actually **depend** on these loadbalancers for proper functionality.